src/js/components/agency/overview/PageTitle.jsx

```diff
@@ -33,13 +28,6 @@ const PageTitle = ({
             <div className="page-title__wrapper">
                 <h2 className="page-title__name">
                     {name}&nbsp;&nbsp;
-                    {name && covidDefCodes.length > 0 &&
-                    <TooltipWrapper tooltipPosition="bottom" offsetAdjustments={{ top: 0 }} className="page-title__tooltip" tooltipComponent={<CovidTooltip fy={fy} codes={covidDefCodes} />}>
```

Comment:
There is code in _pageTitle.scss that can be removed too. For the page-title__tooltip class, and for tooltip-wrapper, tooltip-spacer, and tooltip-pointer.

Reply:
thanks, removing now